changelogs/fragments/proxy.yml

```diff
@@ -0,0 +1,18 @@
+breaking_changes:
+  - zabbix_server_host - Removed zabbix_server_host and replaced with zabbix_proxy_server
```

Comment:
This format assumes that each chnagelog fragment will be assigned correct reference to module or role. Unfortunately, this is not the case and the resulting changlog format will be hard to read. 

That is why we prepend each line with 
```
  - zabbix_proxy (role) - Removed zabbix_server_host and
```